src/editors/containers/LibraryContentEditor/utils.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * checks if library id is v1. defaults to false
+ * @param {[string]} libraryId - target library id
+ * @returns {[boolean]} true if library id is v1
+ */
+export const isV1Library = (libraryId) => {
+  if (!libraryId || (typeof libraryId !== 'string')) {
+    return false;
+  }
+  if (libraryId.split(':')[0] === 'library-v1') {
+    return true;
+  }
+  return false;
+};
+
+/**
+ * gets the name of a library whether it is v1 or v2
+ * @param {[object]} library - library
+ * @returns {[string]} library name. returns an empty string if nothing is found
+ */
+export const getLibraryName = (library) => {
+  if (!library) {
+    return '';
+  }
+  return library.title || library.display_name || '';
```

Comment:
will object has no attribute errors be a problem here?

Reply:
If there's no attribute, it should return `undefined` and go to the next item. This should essentially be
```
if (library.title) return library.title;
if (library.display_name) return library.display_name;
return '';
```